apps/question/urls.py

```diff
@@ -12,4 +11,7 @@
     path('api/attempt/<int:attempt_id>/<int:question_id>/',
          QuestionAnswerView.as_view(),
          name='question_answers_api'),
+    path('api/attempt/<int:attempt_id>/<int:question_id>/flag',
```

Comment:
It seems this endpoint belongs to the /attempt/ path instead of question. I'd move it there.